src/main/resources/application.yml

```
@@ -0,0 +1,14 @@
+    url: jdbc:mysql://db:3306/bitly_db?useSSL=false&serverTimezone=UTC&characterEncoding=UTF-8
+    username: root
+    password: root
+    driver-class-name: com.mysql.cj.jdbc.Driver
+  jpa:
+    hibernate:
+      ddl-auto: update
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true
+        dialect: org.hibernate.dialect.MySQLDialect
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)